docs/Users_Guide/tc-pairs.rst

```diff
@@ -7,7 +7,7 @@ TC-Pairs Tool
 Introduction
 ============
 
-The TC-Pairs tool provides verification for tropical cyclone forecasts in ATCF file format. It matches an ATCF format tropical cyclone (TC) forecast with a second ATCF format reference TC dataset (most commonly the Best Track analysis). The TC-Pairs tool processes both track and intensity adeck data and probabilistic edeck data. The adeck matched pairs contain position errors, as well as wind, sea level pressure, and distance to land values for each TC dataset. The edeck matched pairs contain probabilistic forecast values and the verifying observation values. The pair generation can be subset based on user-defined filtering criteria. Practical aspects of the TC-Pairs tool are described in :numref:`TC-Pairs_Practical-information`. 
+The TC-Pairs tool provides verification for tropical cyclone forecasts in ATCF file format. It matches an ATCF format tropical cyclone (TC) forecast with a second ATCF format reference TC dataset (most commonly the Best Track analysis). The TC-Pairs tool processes both track and intensity adeck data and probabilistic edeck data. The adeck matched pairs contain position errors, as well as wind, sea level pressure, and distance to land values for each TC dataset. The edeck matched pairs contain probabilistic forecast values and the verifying observation values. The pair generation can be subset based on user-defined filtering criteria. Practical aspects of the TC-Pairs tool are described in :numref:`TC-Pairs_Practical-information`.
```

Comment:
What is meant by "wind" in this sentence? If it is actually storm intensity, I suggest changing "wind" to "intensity". But if it's something different (like wind radii), then it'd be good to clarify that as well. 

Similarly, is "sea level pressure" actually "minimum sea level pressure" (the lowest sea level pressure near the storm center)? If so, I'd suggest changing this as well.

Reply:
Thanks for the clarification @jvigh.

Changing the wording...
FROM: `The adeck matched pairs contain position errors, as well as wind, sea level pressure, and distance to land values for each TC dataset.`
TO: `The adeck matched pairs contain position errors, as well as values for each TC dataset, including wind speed, wind radii, minimum sea level pressure, and distance to land.`